AWS account signup (#3891)
timescale/docs: _partials/_cloud-installation.md

## Create a $CLOUD_LONG account

You create a $CLOUD_LONG account to manage your $SERVICE_SHORT and data in a centralized and efficient manner in $CONSOLE. From $CONSOLE_SHORT, you can create and delete $SERVICE_SHORTs, run queries, manage access and billing, integrate other services, contact support, and more.

<Tabs label="Create a Timescale Cloud account">

> <Tab title="Standalone">

Reviewer: ```suggestion
<Tab title="$CONSOLE">
```
That is, if the variable works.

Author: No, the variable does not work. My concern is that in 1. both cases you create an account in Timescale Console, you just start from AWS in one case. 1. I use "standalone" not only in the tab.

Reviewer: Standalone makes me think of self-hosted TimescaleDB, rather than the Cloud solution. I guess you could say Timescale billing and AWS billing or something like that.

You create a standalone account to manage $CLOUD_LONG as a separate unit in your infrastructure, which includes separate billing and invoicing. 

To create an account:

<Procedure>

> 1. **Click `Try for free` on [timescale.com][timescale-website]**

Reviewer: Remove

1. **Sign up for a 30-day free trial** 

>     You receive a confirmation email in your inbox.

Reviewer: Open [Sign up for Timescale][trial-signup] and add your details, then click `Sign up for Timescale`.

[trial-signup]: https://console.cloud.timescale.com/signup

1. **In the confirmation email, click the link supplied and sign in to [$CONSOLE][tsc-portal]**

1. **Answer the requirements questions**

    Your answers help us optimize $SERVICE_LONGs for your use cases.

</Procedure>

</Tab>

<Tab title="AWS Marketplace">

You create an account through AWS Marketplace to have $CLOUD_LONG as a part of your AWS infrastructure. In this case, $CLOUD_LONG is a line item in your AWS invoice. 

To create an account:

<Procedure>

1. **Open [AWS Marketplace][aws-marketplace] and search for `Timescale Cloud`** 

   You see two pricing options, [pay-as-you-go][aws-paygo] and [annual commit][aws-annual-commit]. 

> 1. **Select the right pricing option and click `View purchase options`**

Reviewer: ```suggestion
1. **Select the pricing option that suits you and click `View purchase options`**
```

1. **Review and configure the purchase details, then click `Subscribe`**

1. **Click `Set up your account` at the top of the page**

   You are redirected to $CONSOLE.

1. **Sign up for a 30-day free trial**

>    If you want to link your existing $CLOUD_LONG account to AWS, log in with your existing credentials.  

Reviewer: ```suggestion
   If you want to link an existing $CLOUD_LONG account to AWS, log in with your existing credentials.  
```

> 1. **Click `Connect` in `Confirm AWS Marketplace connection`**

Reviewer: ```suggestion
1. **In `Confirm AWS Marketplace connection`, click `Connect` **
```
